bot/cogs/sync/syncers.py

```diff
@@ -191,24 +50,8 @@ async def sync(self, guild: Guild, ctx: t.Optional[Context] = None) -> None:
         """
         log.info(f"Starting {self.name} syncer.")
 
-        message = None
-        author = self.bot.user
-        if ctx:
-            message = await ctx.send(f"📊 Synchronising {self.name}s.")
-            author = ctx.author
-
+        message = await ctx.send(f"📊 Synchronising {self.name}s.") if ctx else None
```

Comment:
Personal opinion here so I'd like to hear other opinions, but I don't like the send in a ternary too much; with something like this or message initialized before the check to None being clearer to me
```suggestion
        if ctx:
            message = await ctx.send(f"📊 Synchronising {self.name}s.")
        else:
            message = None
```

Reply:
The suggestion is just as clear, if not clearer, so I don't think this needs much discussion.